src/App.tsx

```diff
@@ -19,13 +27,15 @@ export const App: React.FC = () => {
                 type="text"
                 id="search-query"
                 className="input"
+                value={query}
                 placeholder="Type search word"
+                onChange={event => setQuery(event.target.value)}
```

Comment:
it's better to create a separate function `handleChange` instead of passing an anonymous function